test/components/urlBarTest.js

```diff
@@ -89,18 +88,11 @@ describe('urlBar', function () {
         })
     })
 
-    it('it does not change input after focus until keydown', function * () {
-      yield this.app.client
-        .keys('https://brave.com/ ')
-        .keys('\uE007')
-        .waitForElementFocus(activeWebview)
-        .ipcSend('shortcut-focus-url')
-        .getValue(urlInput).then((val) => assert(val === 'https://brave.com/'))
-        .keys('\uE015')
+    it('does not show suggestions on focus', function * () {
```

Comment:
i think you want to type some letters before focusing, otherwise it wouldn't show suggestions anyway